src/ws_correction.py

```diff
@@ -8,6 +8,7 @@
 
 from src.lib import PipelineConfiguration
```

Comment:
Let me know if any of this file is different to OCHA

Reply:
Just a slight modification regarding SENT_ON key
https://github.com/AfricasVoices/Project-IMAQAL/blob/823a6746d2af9878febb30be36bac3d42d085661/src/ws_correction.py#L237